### ISSUE TITLE
An example that re-uses existing components in custom field

### ISSUE THREAD

**Issue by pdonchev** [Sitefinity/sitefinity-admin-app-extensions#109]
As a developer I would like to be able to re-use existing component (such as image selector for example) in order to create more complicated field selectors.

**Comment by pdonchev** [comment]
Any updates on this?

**Comment by mrambold** [comment]
I agree, this would be great. We would for example like to use the existing page picker or related data picker in one of our custom fields.
